frontend/src/features/layout/Sidebar/navigation/NavigationMenuList.scss

```
@@ -0,0 +1,147 @@
+@import '../../../../static/css/includes/_variables_dimensions.scss';
```

Stylelint (17.11.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)